### DISCUSSION TITLE
讓網路科學工具助你成為劇透魔人，資料科學家建立《獵魔士》人際網路圖

### DISCUSSION THREAD

**Discussion by yucheng1028** [1101-GS4719-C/Python-Community-GS4719-C#434] [General]
身為玩過＜巫師３＞及收看＜獵魔人＞兩季的粉絲，看到現今能使用網路科學（Network Science），能完整畫出一般人很難記得之人物角色之間的關係圖，使未來的網路資料收集，能同步歸納分析真實世界之資訊，讓科學家能利用這些工具在學術資料整理及蒐集上，使效率能大大的提升。

https://technews.tw/2022/03/03/data-scientist-builds-a-detailed-network-map-of-the-witcher/

**Comment by JSHT** [comment]
Just 提醒，這篇已經有人發過囉
[discussion 426](https://github.com/NCU-GS4719-Python/Python-Community/discussions/426)
